src/gateways/PostgreUtilisateurLoader.ts

```diff
@@ -39,6 +44,24 @@ export class PostgreUtilisateurLoader implements MesUtilisateursLoader {
         // @ts-expect-error
         where = { ...where, role: { in: roles } }
       }
+
+      if (codeDepartement !== departementInexistant) {
+        where = { ...where, departementCode: codeDepartement }
```

Comment:
Pour simplifier l'écriture, je propose : 
```suggestion
        where.departementCode = codeDepartement
```
En plus ça permet de ne muter qu'une propriété de l'objet et pas l'objet en entier